src/scf-release/jobs/install-suse-fuse/spec

```diff
@@ -0,0 +1,9 @@
+---
+name: install-suse-fuse
+description: Install the SUSE userspace FUSE support.
+
+templates:
+  pre-start.erb: bin/pre-start
+
+packages:
+  - suse-fuse
```

Comment:
No need to indent list elements:

```
packages:
- suse-fuse
```

Reply:
Done with commit f5ad84cb4a10919001addd629b9cafd7c85bd1a1